resources/routes/list.ts

```diff
@@ -3,17 +3,19 @@
 /// file COPYING or read https://opensource.org/licenses/MIT
 
 /// Import All route from here.
-import LoginView from '../app/login/Login';
+import LoginView from '@app/login/Login';
 
 export default [
     {
         path: "/",
         auth: true,
         component: LoginView,
+        exact: true,
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
    exact: true,
```